reaper.go

```diff
@@ -126,8 +126,14 @@ func (r *Reaper) Connect() (chan bool, error) {
 		for retryLimit > 0 {
 			retryLimit--
 
-			sock.WriteString(strings.Join(labelFilters, "&"))
-			sock.WriteString("\n")
+			if _, err := sock.WriteString(strings.Join(labelFilters, "&")); err != nil {
+				continue
+			}
+
+			if _, err := sock.WriteString("\n"); err != nil {
+				continue
+			}
```

Comment:
@mdelapenya Can you confirm this is okay? Tests pass, but I'd like to be extra careful here.

Reply:
Given in https://github.com/testcontainers/testcontainers-go/pull/569/files#diff-e8ab4ec17e978db949cd15726ba0efa9c89fd714e7e6e0aa537ed7fc1bbde7fdR138 we continue on a flush failure, I'd say that it's okay